uber/tests/models/test_attendee.py

```diff
@@ -321,6 +321,13 @@ def yes_to_no_volunteering(self, unset_volunteering):
             assert unset_volunteering.called
 
 
```

Comment:
Two minor style notes about this test:
- For the sake of consistency, you should make it part of the ``TestStaffingAdjustments`` class above.
- You should probably just pass ``ribbon`` as a keyword parameter along with ``first_name`` and ``last_name``, since there's no need to have a separate line for it.

Those are just style concerns though.  Good test!